scripts/script.js

```diff
@@ -0,0 +1,22 @@
+
+window.onload=function(){
+    document.getElementById("menu-hamburger").addEventListener("click", toggleMenu);
+}
+
+const toggleMenu = () => {
+    var navBar=document.getElementById("navigation");
+    console.log("clicked")
+    if (navBar.classList.contains("nav-mobile")) {
+        navBar.classList.remove("nav-mobile");
+        navBar.classList.add("navigation");
+    } else {
+        navBar.classList.remove("navigation");
+        navBar.classList.add("nav-mobile");
+    }
+    window.addEventListener("resize", function(event) {
+    if (navBar.classList.contains("nav-mobile")) {
+        navBar.classList.remove("nav-mobile");
+        navBar.classList.add("navigation");
+    }
+})
```

Comment:
You are adding a new 'resize' event listener every the user clicks on the hamburger button, which will generate too many events (you can test this by adding a debugger statement inside the resize listeners and see how many it gets called after clicking on the hamburger button).

You should move this listener inside the 'onload' function instead.

Reply:
Fixed it now, not sure how I missed that!

Thanks as always!